### ISSUE TITLE
Shake.yaml file not found

### ISSUE THREAD

**Issue by awgn** [haskell/haskell-language-server#162]
When building using stack:

```
stack ./install.hs hls
```

I get the following error:

```
# stack (for hls-8.8.3)
Could not parse '/Users/nicola/3rdParty/haskell-language-server/install/shake.yaml':
YAML exception:
Yaml file not found: /Users/nicola/3rdParty/haskell-language-server/install/shake.yaml
See http://docs.haskellstack.org/en/stable/yaml_configuration/
```

**Comment by fendor** [comment]
Thank you for your bug report and sorry for the inconvenience! 
A fix is proposed in #163

**Comment by jneira** [comment]
I did the change in the first place and my local tests after removing `shake.yaml`worked for some reason i have to investigate. The fix is already in master, thank to quick @fendor fix.
